lib/helpers/tasks.rb

```diff
@@ -1,7 +1,6 @@
 require 'open3'
 require 'slack-notifier'
 require 'mcollective'
-require 'io'
```

Comment:
@dhollinger Not sure this is right!!  But I wanted to get the tests running.  PR is WIP :)